### DISCUSSION TITLE
Siterwell GS361A-H04 - no valve position value

### DISCUSSION THREAD

**Discussion by GrohmS2977** [Koenkk/zigbee2mqtt#10621] [Devices]
Hello everyone,

yesterday i installed my first heating thermostat, an Essentials 120112 (white label for Siterwell GS361A-H04 (this one: https://www.zigbee2mqtt.io/devices/GS361A-H04.html#siterwell-gs361a-h04))

I wonder why i don't get any value for valve position and running_state. Is it not supported by this valve or am i doing something wrong? 

I got Home Assistant running on a Raspi with CC2531 dongle, zigbee2mqtt latest release 1.22.2.

Seems i'm not the only one with that problem: https://issueexplorer.com/issue/Koenkk/zigbee2mqtt/8813

```
{
    "away_mode": "OFF",
    "battery": 98,
    "child_lock": "UNLOCK",
    "current_heating_setpoint": 18,
    "linkquality": 31,
    "local_temperature": 24.5,
    "position": null,
    "preset": "schedule",
    "running_state": null,
    "system_mode": "auto",
    "valve_detection": "ON",
    "window_detection": "ON"
}
```
Hope you can help me...

Thanks!

Silvio

**Comment by lingi-62** [comment]
Have the same issue .... all working well, but valve position and  running state are not returned.

Does anybody have a solution?

Thanks

Werner

**Comment by ovizii** [comment]
Just wanted to add I have the same problem/behaviour.

**Comment by BiboXXX** [comment]
+1 ... Same problem.

**Comment by s0fax** [comment]
+1

**Comment by MatzeP** [comment]
I have the same problem.
My HA is Up To Date.

**Comment by lastfreeusernameicouldfind** [comment]
+1 same problem. Can someone please help?
